en-us/multimc-auto-update.md

```diff
@@ -36,44 +36,49 @@ You'll have to download a new version of the pack that is specific to that Minec
+1. Open a terminal and run the command below:
+   * MacOS: `brew install jq`
+   * Arch Linux: `sudo pacman -S jq`
+   * Debian-Ubuntu Linux: `sudo apt-get install jq`
+   * Fedora Linux: `sudo dnf install jq`
+   * openSUSE Linux: `sudo zypper install jq`
 
-![Screenshot-1](https://user-images.githubusercontent.com/77157639/156615703-f113293c-e821-4c94-a891-2fccd0ff8848.png)
+This will install a program called [jq](https://stedolan.github.io/jq/), needed for automatically adapting this script to any Minecraft version you use.
 
-Now, inside the terminal paste the command below:
+If don't use any of the distros above, you have to follow [this](https://stedolan.github.io/jq/download/) tutorial.
 
-`curl -Os https://gist.githubusercontent.com/Remty5/1b8a8d7c842dda56dacebf6db8c10961/raw/c0149c39ef4be375d013fa254b71899fb4bd6105/post-exit.sh && curl -Os https://gist.githubusercontent.com/Remty5/1b8a8d7c842dda56dacebf6db8c10961/raw/c0149c39ef4be375d013fa254b71899fb4bd6105/pre-launch.sh`
+2. Download the mod disabling scripts:
+   * Open MultiMC, right click your instance and click "Instance Folder"
+   * Inside the folder now, right click and select "Open Terminal here" (to access this option in MacOS, follow [this](https://www.petenetlive.com/KB/Article/0001060) tutorial)
+   * Within the terminal, run the following command - this will install the files and make them executable:\
 
-If everything goes well, the scripts are installed!
+`curl -Os https://raw.githubusercontent.com/RaptaG/fabulously-optimized/main/Packwiz/pre-launch.sh | curl -Os https://raw.githubusercontent.com/RaptaG/fabulously-optimized/main/Packwiz/post-exit.sh && chmod +x pre-launch.sh post-exit.sh`
 
-Now there are 4 things remaining:
+ If you want to install them somewhere else, open up a terminal and run `cd /path/to/folder` (replace `/path/to/folder` with the folder's location), before running the command above.
 
-1. upgrade the checksums,
-2. actually select which mods you want to ignore,
-3. upgrade the `pack.toml`
-4. make them work in MultiMC
+3. Select the mods to disable
+   1. Copy the name of the mod(s) you want to disable
+   2. Open `pre-launch.sh` with any text editor
+   3. Find the line saying "Select the mods you wish to disable:" and below it `mod0=`, `mod1=`, `mod2=`,`mod3=`, `mod4=` and `mod5=`. After `=` place the name of the mods you previously copied, one by one.
+   * Note: No matter how many mods you disable, **never** remove `mod0=` and `$mod0.jar`
 
-*   Within the terminal, run another command:
+4. Setup the scripts to run on your MultiMC (auto-update) instance  
+   1. Open MultiMC
+   2. Click on your instance "Edit Instance"
+   3. Go to "Settings" and then to "Custom Commands"
+   4. Remove the pre-launch command and replace it with `/path/to/folder/pre-launch.sh` (again, replace `/path/to/folder` with the folder where you              installed the scripts
```

Comment:
This too

Reply:
> This too, and the one below it

https://github.com/RaptaG/wiki/commit/34ca8855225569c985b495751b15cb6c81529fde